client/README.md

```diff
@@ -163,6 +163,8 @@ Debugging your smart contract allows you to run through the smart contract trans
 
 To make iterative changes to your smart contract while debugging, after making your changes click the **restart** button. Restarting debugging means you don't need to instantiate the contract again. Please note, as this stores the smart contract in local memory, for many changes to large smart contracts, you may need to reinstantiate the smart contract.
 
+If you stop the fabric runtime, in order to debug you’ll need to rename your smart contract or teardown the fabric runtime.
```

Comment:
i think you need to say something like `If you stop the fabric runtime after a debug session, you will need to rename your smart contract  or teardown the fabric runtime and restart it to continue the debug session. This is because when fabric starts up again it will automatically create the chaincode container.`